simple-weather/simple-weather/Info.plist

```diff
@@ -49,5 +49,10 @@
 	<string>we need your location to show weather !</string>
 	<key>LSApplicationCategoryType</key>
 	<string></string>
+	<key>NSAppTransportSecurity</key>
+	<dict>
+		<key>NSAllowsArbitraryLoads</key>
```

Comment:
`UI` branch has networking, result of making branch of `networking` branch instead of `develop` branch